presto-main/src/main/java/io/prestosql/sql/ExpressionUtils.java

```diff
@@ -225,7 +233,35 @@ public static Expression combineDisjunctsWithDefault(Metadata metadata, Collecti
             return TRUE_LITERAL;
         }
 
-        return disjuncts.isEmpty() ? emptyDefault : or(disjuncts);
```

Comment:
shouldn't this rewrite be part of `io.prestosql.sql.planner.iterative.rule.CanonicalizeExpressions` or `io.prestosql.sql.planner.iterative.rule.SimplifyExpressions` instead? @martint ?

Reply:
Yes, that would be preferable.